bin/apisix

```diff
@@ -21,6 +21,8 @@ local function trim(s)
     return (s:gsub("^%s*(.-)%s*$", "%1"))
 end
 
+-- Note: The `excute_cmd` return value will have a line break at the end,
+-- it is recommended to use the `trim` function to handle the return value.
```

Comment:
we can move this comment to https://github.com/apache/incubator-apisix/pull/1658/files#diff-62edadffc237f13dc28a694080d293a8R670

Reply:
I think it's reasonable to put this function usage note in the corresponding function definition.

Reply:
ok